server/test/controllers/admin/AdminProgramTranslationsControllerTest.java

```diff
@@ -227,6 +230,7 @@ public void update_validationErrors_rendersEditFormWithMessages()
                 ImmutableMap.<String, String>builder()
                     .put("displayName", "")
                     .put("displayDescription", "")
```

Comment:
Include empty `displayDescription` so can test that no error message comes through for that